.github/workflows/partial-release.yml

```diff
@@ -0,0 +1,96 @@
+name: Release
+
+on:
+  workflow_call:
+    inputs:
+      draft:
+        type: boolean
+        required: true
+      tag:
+        type: string
+        required: true
+
+jobs:
+  release:
+    name: Release CLI
+    runs-on: buildjet-8vcpu-ubuntu-2004
```

Comment:
Can we use these with an open source repo? Would it change pricing? (we currently don't pay for CI runs in this repo)

Reply:
good point, it's paid. Any reason we're avoiding costs?

Reply:
Not specifically, but I'd ask Fredrik before changing this to paid